cppexpat.hpp

```diff
@@ -43,20 +43,26 @@ namespace cppexpat {
     class XMLError: public std::exception {
     public:
         inline XMLError(XML_Parser p):
-            msg{XML_ErrorString(XML_GetErrorCode(p))},
-            lineno{XML_GetCurrentLineNumber(p)},
-            colno{XML_GetCurrentColumnNumber(p)} {}
+            what_(construct_what(p))
```

Comment:
@siebenschlaefer Pretty much. I probably need to document it...somewhere...at some point in time...